.circleci/config.yml

```diff
@@ -74,6 +74,20 @@ commands:
           webhook: $SLACK_GREENKEEPER_WEBHOOK
 
 jobs:
+  auto-rebase:
+    docker:
+      - image: boltdev/m2-installed-plugin-ci-php72:2.3.0-v2
+        auth:
+          username: $DOCKERHUB_USER
+          password: $DOCKERHUB_PASS
+    steps:
+      - checkout
+      - run:
+          name: Auto-rebase
+          command: |
+            ls -al .circleci/scripts/auto-rebase.sh
```

Comment:
Do we need the LS still?

Reply:
removed, it was originally for debugging purpose